docs/security-best-practices.md

```diff
@@ -153,6 +153,108 @@ To change the default endpoint to be `localhost`-bound in all components, enable
 
 If `localhost` resolves to a different IP due to your DNS settings then explicitly use the loopback IP instead: `127.0.0.1` for IPv4 or `::1` for IPv6. In IPv6 setups, ensure your system supports both IPv4 and IPv6 loopback addresses to avoid issues.
 
+Using `localhost` may not work in environments like Docker, Kubernetes, and other environments that have non-standard networking setups.  We've documented a few working example setups for the OTLP receiver gRPC endpoint below, but other receivers and other Collector components may need similar configuration.
+
+#### Docker
+You can run the Collector in Docker using a configuration like the following:
```

Comment:
This sentence makes it feel like the YAML below is a fully working configuration file for the Collector, but it is not. Can we make that clear or make a fully working example?

Reply:
I'll keep them as all as partial configurations and try to clarify that.  We have partial configurations all over the documentation and I think we should stay with that course